tf_files/aws/modules/management-logs/README.md

```diff
@@ -0,0 +1,93 @@
+# TL;DR
+
+Centralize logging for all commons in other AWS accounts
+
+## 1. QuickStart
+
+```
+gen3 workon csoc management-logs
+```
+
+## 2. Table of Contents
+
+- [1. QuickStart](#1-quickstart)
+- [2. Overview](#2-overview)
+- [3. Centralized  logging](#3-centralized-logging)
+  - [3.1 How it works](#21-how-it-works)
+    - [3.1.1 JSON breakdown](#311-json-breakdown)
+
+
+## 2. Overview
+
+The CSOC account is conceptualized to represent a centralized management account for all commons account. The commons account, conceptually, would act like child accounts for CSOC.
+
+
+
+## 3. Centralized logging
+
+For each account, all management logs collected by cloud-trail will be send to CloudWatchLogs
+
+This CSOC account will also collect all those logs generated by its child accounts, for a more easy way to browse them all if required without needing to log into each account.
+
+As for now, logs will be send to a bucket. We can later send them to ES if we wanted to.
```

Comment:
It looks like the logs are being set to ES too?  Can we make that optional in the terraform code based on input variables?